src/redux/books/books.js

```diff
@@ -1,7 +1,7 @@
 const ADD_BOOK = 'ADD_BOOK';
 const REM_BOOK = 'REM_BOOK';
 
-let nextId = 0;
+let nextId = 3;
 
 export function addBookObject(title, author) {
   nextId += 1;
```

Comment:
- [ ] (Optional) It would be nice to install `UUID` and use it to generate a unique ID for each book object. Using `UUID` makes the id very unique. You can read their [documentation](https://www.npmjs.com/package/uuid) to have an understanding of what it entails. 👍